docs/inferences/getting-started.md

```diff
@@ -0,0 +1,39 @@
+
+Once you are ready to [acquire images](image-acquisition/image-acquisition.md), it's now time to perform inferences on them. This section will cover the basics of how to perform inferences on images to classify, detect objects or create segmentation masks on the acquired frames.
+
+## Building your first model
+
+To run inferences using LandingLens, you must first build a model. If you didn't sign up before, visit https://app.landing.ai/, sign up for a free account and create a new project. If you are not familiar with LandingLens, you can find a lot of useful information in the [LandingLens support center](https://support.landing.ai/docs/landinglens-workflow).
+
+Long story short, after creating a project in LandingLens you will need to:
+
+1. Upload example images
+2. Label those images
```

Comment:
the example images are already label right?

Reply:
Hum... I think the term "examples" might be misleading, confusing with the dataset samples we have in the platform. Let me change that a bit.